LayoutTestBase/UIViewHelpers/UIView+LYTFrameComparison.m

```diff
@@ -27,10 +27,35 @@ - (BOOL)lyt_before:(UIView *)otherView {
     }
 }
 
+- (BOOL)lyt_before:(UIView *)otherView fromCenter:(BOOL)fromCenter {
+    CGRect otherViewBounds = [self convertRect:otherView.bounds fromView:otherView];
+    CGFloat epsilon = [LYTConfig sharedInstance].cgFloatEpsilon;
+    
+    if ([self lyt_leftToRight]) {
+        if (fromCenter) {
+            return self.center.x <= otherViewBounds.origin.x + epsilon;
+        }
+        else {
+            return self.bounds.origin.x + self.bounds.size.width <= otherViewBounds.origin.x + epsilon;
+       }
```

Comment:
nitpick: here and below put `else` on same line as `}`

Reply:
Fixed